indico/core/db/sqlalchemy/custom/int_enum.py

```diff
@@ -65,14 +67,24 @@ def coerce_set_value(self, value):
         return self.enum(value)
 
     def alembic_render_type(self, autogen_context):
+        name = '_{}'.format(self.enum.__name__)
+        members = '\n'.join(['    {} = {}'.format(x.name, x.value) for x in self.enum])
+        enum_tpl = textwrap.dedent("""
+            # ### code auto-generated by PyIntEnum - please adjust
+            from enum import Enum
```

Comment:
Can you move this into a separate `imports` entry? That way it doesn't get duplicated in case more than one such column is used in the same revision.

Reply:
Damn, this breaks the file because now the imports is after the code... Anyway let's see if there is a nicer way (as mentioned on IRC)